web/src/styles.css

```
@@ -1017,32 +1017,32 @@ button.work-table-row:hover { background: var(--panel-2); color: var(--fg); }
-  .work-table-row > :not(:first-child):not(:fourth-child) { display: none; }
-  .work-clean-card { grid-template-columns: 90px minmax(160px, 1fr); }
-  .work-clean-card > :nth-child(3), .work-clean-card > :nth-child(4) { display: none; }
-  .target-row summary { grid-template-columns: minmax(160px, 1fr) 90px; }
-  .target-time { display: none; }
+  .overview-facts, .run-summary-strip { grid-template-columns: 1fr 1fr; }
+  .task-row { grid-template-columns: 1fr auto; }
+  .task-schedule { display: none; }
+  .task-actions .icon-button { display: none; }
+  .task-settings, .repository-picker { grid-template-columns: 1fr; }
+  .run-table-row { grid-template-columns: minmax(160px, 1fr) 100px; }
+  .run-table-row > :not(:first-child):not(:fourth-child) { display: none; }
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":fourth-child" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)